aurelia-slickgrid/src/aurelia-slickgrid/editors/multipleSelectEditor.ts

```diff
@@ -0,0 +1,178 @@
+import { arraysEqual } from '../services/index';
+import {
+  Editor,
+  Column,
+  MultipleSelectOption,
+  SelectOption
+} from './../models/index';
+import * as $ from 'jquery';
+
+/**
+ * Slickgrid editor class for multiple select lists
+ */
+export class MultipleSelectEditor implements Editor {
+  /**
+   * The JQuery DOM element
+   */
+  $editorElm: any;
+  /**
+   * The slick grid column being edited
+   */
+  columnDef: Column;
+  /**
+   * The multiple-select options for a multiple select list
+   */
+  defaultOptions: MultipleSelectOption;
+  /**
+   * The default item values that are set
+   */
+  defaultValue: any[];
+  /**
+   * The options label/value object to use in the select list
+   */
+  collection: SelectOption[] = [];
+  /**
+   * The property name for values in the collection
+   */
+  valueName: string;
+  /**
+   * The property name for labels in the collection
+   */
+  labelName: string;
+
+  constructor(private args: any) {
+    this.defaultOptions = {
+      container: 'body',
+      filter: false,
+      maxHeight: 200,
+      width: '100%',
```

Comment:
you should not fix this to 100%, it makes the option "autoDropWidth" unusable. The "autoDropWidth" is a new option that I added which will resize the drop menu with the same size as the input select element.